packages/aws-cdk/lib/legacy-types.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * @deprecated
  */
-export type Obj<T> = {[key: string]: T};
+export type Obj<T> = { [key: string]: T };
```

Comment:
This wasn't picked up before because the default eslint `key-spacing` rules doesn't do TypeScript.